dubbo-config/dubbo-config-spring/src/main/java/org/apache/dubbo/config/spring/ReferenceBean.java

```diff
@@ -181,19 +181,19 @@ public void afterPropertiesSet() throws Exception {
 
         if (getMetadataReportConfig() == null) {
             Map<String, MetadataReportConfig> metadataReportConfigMap = applicationContext == null ? null : BeanFactoryUtils.beansOfTypeIncludingAncestors(applicationContext, MetadataReportConfig.class, false, false);
-            if (metadataReportConfigMap != null && metadataReportConfigMap.size() == 1) {
+            if (CollectionUtils.mapSizeEq(metadataReportConfigMap, 1)) {
                 // first elements
                 super.setMetadataReportConfig(metadataReportConfigMap.values().iterator().next());
-            } else if (metadataReportConfigMap != null && metadataReportConfigMap.size() > 1) {
+            } else if (CollectionUtils.mapSizeGt(metadataReportConfigMap, 1)) {

```

Comment:
I do not see the benefits of this change, actually it increase the difficulty to understand. I'd rather keep the original logic.